test-case/check-alsabat.sh

```diff
@@ -58,6 +58,15 @@ then
 	exit 2
 fi
 
+# If DEVICE0_MODEL is defined, set proper gain for the platform
+if [ -z "$DEVICE0_MODEL" ]; then
```

Comment:
`DEVICE0_MODEL` is what we set in DUT env setup. But now a device may be selected under multiple MODEL condition, so it may not be correct. for example, we are running a apl_nocodec_zephyr test, but the value for `DEVICE0_MODEL` is apl_nocodec

Reply:
Good point, we don't need to check ZEPHYR separately. ZEPHYR platforms use same tplg, I clean up ZEPHYR